Delete compiler options not available in release versions of clang _yet_

Upstream-Status: Inappropriate [ clang/master already supports them ]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1518,6 +1518,11 @@ config("default_warnings") {
         # Flags NaCl (Clang 3.7) and Xcode 9.2 (Clang clang-900.0.39.2) do not
         # recognize.
         cflags += [
+          # Disable unknown warnings, since system clang may not have
+          # backported newer warning yet unlike internal clang that
+          # chromium uses.
+          "-Wno-unknown-warning-option",
+
           # An ABI compat warning we don't care about, https://crbug.com/1102157
           # TODO(thakis): Push this to the (few) targets that need it,
           # instead of having a global flag.
--
2.25.1
